Corona-Warn-App/src/main/res/layout/fragment_settings_tracing.xml

```diff
@@ -92,35 +92,54 @@
                     app:statusText="@{settingsTracingState.getTracingStatusText(context)}"
                     app:subtitle="@{@string/settings_tracing_title}" />
 
-                <androidx.appcompat.widget.LinearLayoutCompat
-                    android:id="@+id/settings_interoperability_row"
-                    style="@style/row"
-                    android:layout_width="match_parent"
+                <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Although the performance was ok on my Pixel, this introduced at third-level nested Constraint Layout

Reply:
Thanks! - Yes, I am aware of this. However, I haven't found a better solution without refactoring the whole layout. Do you have an idea?